### ISSUE TITLE
default css is loaded despite clearing the field in the plugin configuration

### ISSUE THREAD

**Issue by Aero91** [teaminmedias-pluswerk/ke_search#411]
Hi,
I just updatet from version 3.4.0 to 3.6.1. 
Unfortunately, the extensions default css file is now loaded despite removing it from the plugin configuration. It worked just fine prior to the update. 

TYPO3 version 10.4.14 (composer)

**Comment by christianbltr** [comment]
The way to unset the default CSS has been changed from flexform to typoscript.

> Unset via typoscript
> 
> If you do not wish to load that file, you can unset it via typoscript:
```
plugin.tx_kesearch_pi1.cssFile >
plugin.tx_kesearch_pi2.cssFile >
```

See also 

https://docs.typo3.org/p/teaminmedias-pluswerk/ke_search/master/en-us/Templating/Css.html

and

https://github.com/teaminmedias-pluswerk/ke_search/blob/master/ChangeLog

**Comment by medienbaum** [comment]
It still does not work. Tested on Typo3 10.4.20 with the above mentioned typoscript setup. Using ke_search version 3.9.0.

**Comment by Mbigha** [comment]
I also had this problem with the default css still loading after unsetting via TypoScript as stated above.

In my case, I realized this default CSS was still being loaded by a Typoscript object which was created by copying from the `plugin.tx_kesearch_pi1` typoscript object. For example
```
lib.searchbox = COA_INT
lib.searchbox {
    10 < plugin.tx_kesearch_pi1
}
```

So, to fix this, I unsetted the CSS file before copying from the `plugin.tx_kesearch_pi1` object.
